apps/api/src/domain/models/term/commands/create-term/term-created.event-handler.ts

```diff
@@ -11,6 +11,7 @@ export class TermCreatedEventHandler implements ICoscradEventHandler {
     ) {}
 
     async handle(event: TermCreated): Promise<void> {
+        console.log('handling TermCreated');
         // TODO use dynamic registration
         // Can we remove this now?
         if (!event.isOfType('TERM_CREATED')) return;
```

Comment:
We should remove this now, as well. It is not the responsibility of the handler to check for this.

Reply:
done